### ISSUE TITLE
when i run i find some wrong

### ISSUE THREAD

**Issue by yanxiaochuan** [fjbarrena/dtrack-cli#1]
Developed by @DogDeveloper. Show me your love here https://twitter.com/DogDeveloper
Error uploading bom Error: Request failed with status code 405
Review your settings ;)

**Comment by fjbarrena** [comment]
Hi @yanxiaochuan 

I probably need more details about your setup, but in any case I tried to reproduce the error in a fresh installation of Dependency Track. These are the steps I followed:

* Install Dependency Track in a server using the latest version and based in Docker. Using the instructions of the documentation of Dependency Track
```
https://docs.dependencytrack.org/getting-started/deploy-docker/
```
* If all it's right, you will be able to access to the following URL: http://localhost:8080 (change with the IP of your server)
* Then, log in and go to the Teams section inside the administration. Edit the "Automation" Team with the following permissions

<img width="1344" alt="Screenshot 2020-09-15 at 10 39 13" src="https://user-images.githubusercontent.com/1855013/93187232-dfebe300-f73f-11ea-896e-cde4fc69917e.png">

* Make sure you have an API Key for the Automation Team, like the following (don't worry, this key is revoked ;))

<img width="466" alt="Screenshot 2020-09-15 at 10 41 52" src="https://user-images.githubusercontent.com/1855013/93187442-1a558000-f740-11ea-8399-fe0039e2ed83.png">

* Now, with the previous API Key, you will be able to use the dtrack-cli following the instructions provided, in my test using Gitlab CI/CD in a NestJS (NPM/NodeJS) project was as follows:

<img width="1103" alt="Screenshot 2020-09-15 at 10 58 22" src="https://user-images.githubusercontent.com/1855013/93189459-73261800-f742-11ea-9168-0a6b79f4c457.png">

* After the pipeline was executed, yeah! It's true, I got a 405 error

<img width="655" alt="Screenshot 2020-09-15 at 11 13 25" src="https://user-images.githubusercontent.com/1855013/93191154-7fab7000-f744-11ea-8843-3c9c4a987b46.png">

* Then, I tried in my computer, and all gone right with the same code, and the same command and the same parameters

<img width="689" alt="Screenshot 2020-09-15 at 11 21 04" src="https://user-images.githubusercontent.com/1855013/93192167-a6b67180-f745-11ea-8a21-37e5ecb5acb1.png">

* So I guess it's a network problem. I checked if the Dependency Track server has visibility inside the CI/CD, and they have visibility. Probably, the CI/CD servers has some restrictions of use?

**Comment by fjbarrena** [comment]
Oh my goodness!!

Yeah, I found it...

If your server variable ends with a slash, you will take this error.

So your server address will be that:

```
http://localhost:8080    <---- RIGHT
```

instead of that:

```
http://localhost:8080/    <--- WRONG
```

I will change the code to have that into account, but for now you can apply this workaround.

Thanks for the issue!

**Comment by fjbarrena** [comment]
Solved in 1.0.12
